client/src/components/Signup/Signup.js

```diff
@@ -116,9 +119,19 @@ export default function Signup() {
                     <label htmlFor="password">Password *</label>
                   </div>
                   <div className="form-group text-center">
-                    <button type="submit" className="btn btn-primary btn-lg">
-                      Sign Up
-                    </button>
+                    {isDisabledButton ? (
+                      <button
```

Comment:
see button html code is duplicated in many places, use the component from react-bootstrap